### PR TITLE
net/freeradius: make use od certificates clearer

### LINKED ISSUE #1004: freeradius using default certificates
Have configured the freeradius plugin to use eap-ttls. 
Have choosen a root and a server certificate from the internal Ca, but when my clients connect they get default certificate with freeradius attributes. See screenshot. 
screenshot 2018-11-20 at 00 45 40
found all files in /usr/loacl/etc/raddb/certs were generated at install time from the freeradius plugin.
found the that usr/loacl/etc/raddb/mods-enabled/eap contains pointers to the files in the cert directory and it was also generated at install time. 
If I add/change useres or clients they get updated correct. What could be the issue here?

freeradius plugin version: 1.8.2
opnsense 18.7.7

Thanks for support
  Thomas

### PR DESCRIPTION
**Pull request by mimugmail**
Closes #1004

### CODE REVIEW

**Comment by joekaybutter1** [comment]
So what exactly am I suppose to be doing?

**Comment by joekaybutter1** [comment]
Update make file

**Comment by joekaybutter1** [comment]
Merge pull request #1004 from opnsense / master

**Comment by mimugmail** [comment]
Are you OK? Is this some GitHub Spam?

File: net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/eap.xml [hunk 1]
```diff
@@ -7,9 +7,9 @@
     </field>
     <field>
         <id>eap.enable_client_cert</id>
-        <label>Validate Certificates</label>
+        <label>Use own Certificates</label>
```
Comment (fichtner) [inline net/freeradius/src/opnsense/mvc/app/controllers/OPNsense/Freeradius/forms/eap.xml]:
```suggestion
        <label>Use own certificates</label>
```

**Comment by mimugmail** [comment]
Merge and lock :)

**Comment by fichtner** [comment]
merged, thanks!

**Pull request merged**